vscode-extension/package.json

```diff
@@ -0,0 +1,94 @@
+{
+	"name": "darklang",
+	"description": "darklang VS Code Extension",
+	"author": "darklang",
+	"version": "0.0.1",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/darklang/dark"
+	},
+	"publisher": "darklang",
+	"categories": [],
+	"keywords": [
+		""
+	],
+	"engines": {
+		"vscode": "^1.74.0"
```

Comment:
Note: when testing this locally, update VS code.

we could work at lowering the requirement here later if relevant